docs/_functions/domain/CAA.md

```diff
@@ -0,0 +1,34 @@
+---
+name: CAA
+parameters:
+  - name
+  - tag
+  - value
+  - modifiers...
+---
+
+CAA adds a CAA record to a domain. The name should be the relative label for the record. Use `@` for the domain apex.
+
+Tag can be one of "issue", "issuewild" or "iodef". Value depends on the tag, and is automatically quoted.
```

Comment:
If tag can only be "issue", "ussuewild" or "iodef", then NormalizeAndValidateConfig() should produce an error if it is anything else.